content/docs/iac/adopting-pulumi/migrating-to-pulumi/from-arm.md

```diff
@@ -25,7 +25,32 @@ If your team has already provisioned infrastructure using Azure Resource Manager
 * [**Import**](/docs/using-pulumi/adopting-pulumi/import/) existing resources into Pulumi in the usual way.
 * [**Convert**](#converting-stacks-and-resources) your deployments to use Pulumi and then incrementally migrate resources.
 
-## Referencing Stack Outputs
+## Why Move Beyond ARM
+
+ARM templates were Azure’s original infrastructure as code solution. But they fall short for today’s engineering teams:
+
+| Feature           | ARM Templates        | Pulumi                        |
+|-------------------|----------------------|-------------------------------|
+| Language          | JSON / YAML          | C# (or Python, TS, Go)        |
+| Reuse             | Limited (copy/paste) | Functions, classes, modules   |
+| Logic & Loops     | Complex expressions  | if / for / switch             |
+| Type Safety       | None                 | Compile-time type checking    |
+| Tooling           | Basic validation     | Full IDE & IntelliSense       |
+| Testing           | Manual or none       | xUnit, NUnit, etc.            |
+| Code Sharing      | Hard                 | NuGet packages & projects     |
+| Debugging         | Deploy & troubleshoot| Step-through in IDE           |
+| Refactoring       | Tedious manual edits | IDE-assisted refactors        |
+| Modularity        | Nested templates     | Real modules & components     |
+
+Pulumi lets you manage Azure infrastructure using your favorite language, with built-in support for CI/CD, reusable components, and testability. For .NET teams especially, Pulumi’s support for C# provides a superior experience over static JSON.
+
+Continue reading to learn how to:
+
+- [Reference ARM outputs in Pulumi programs](https://www.pulumi.com/docs/iac/adopting-pulumi/migrating-to-pulumi/from-arm/#reference-arm-outputs-in-pulumi-programs)
```

Comment:
It ( the linter ) wants * rather than - for this list ( and below )

Reply:
Thanks, Adam! I though bullet points were just -